Servlets/src/main/java/ua/goit/servlets/servlets/Projects.java

```diff
@@ -0,0 +1,46 @@
+package ua.goit.servlets.servlets;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import ua.goit.servlets.dao.CategoriesList;
+import ua.goit.servlets.model.Category;
+import ua.goit.servlets.model.Project;
+
+public class Projects extends HttpServlet {
```

Comment:
Please add something into the name of the class to give a hint that it's a servlet. E.g. `ProjectsServlet`, `ProjectsController`, ...

Otherwise `Projects` and `List<Project>` visually are the same but totally different in their sense